roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -264,12 +281,16 @@ class ResetNeutron(object):
             for agent in reversed(running_agents):
                 self.start_service(agent)
 
+            for iface in interfaces:
```

Comment:
This is raceful, since we don't know that the bridge interfaces have been recreated after starting the neutron-linuxbridge-agent.